wiki/People/The_Team/Beatmap_Nominators/Expectations/en.md

```diff
@@ -9,7 +9,7 @@ As a reminder, Beatmap Nominators are expected to do their best to adhere to the
 - **Act with the community's best interests at heart.** Approach tenuous situations with caution and care, aim to act with sound judgment wherever possible, seek to learn from your mistakes, and at all times make sure you're thinking about the impact your actions might have on the broader community.
 - **Do not exploit or abuse existing systems.** Following the [Ranking Criteria](/wiki/Ranking_Criteria) and not trying to circumvent it is a given, and cases of disagreement should be resolved through a Ranking Criteria proposal. Beatmap Nominators should also not repeatedly exploit bugs or reward systems such as [score](/wiki/Score), [PP](/wiki/Performance_points) or [kudosu](/wiki/Modding/Kudosu).
 - **Be civil, both in and outside the game's direct platform.** Beatmap Nominators are expected to not be excessively antagonistic, inflammatory, or otherwise derogatory toward other members of the osu! community. This especially applies to communities where other users commonly associate them to their account on osu! (e.g. mapping Discord servers or Twitter).
-- **Mod regularly.** Beatmap Nominators are not only responsible for fixing unrankables, but also making the mapsets they're interested in nominating ready for rank, and ensuring their quality. They should always be vigilant and ready to mod thoroughly, just like they would have for their BN application.
+- **Mod regularly.** Beatmap Nominators are expected to ensure and improve not only rankability, but also quality, of maps they nominate. They should be able to thoroughly mod where necessary. Modding done outside of beatmap discussions should still be accessible there as proof (e.g. through logs/summaries).
```

Comment:
```suggestion
- **Mod regularly.** Beatmap Nominators are expected to ensure and improve not only the rankability but also the quality of maps they nominate. They should be able to mod thoroughly where necessary. Modding done outside of beatmap discussions should still be accessible there as proof (e.g. through logs/summaries).
```
Heavy use of commas hinders the fluency a little and is probably grammatically incorrect. Also, adverbs are used after the verb they modify. Not sure about adding the specifiers. I just felt the lack of them while reading.